### ISSUE TITLE
Cerberus Gates of hades is too slow + has an off era static 5s pause afterwards (if it goes off)

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#5158]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

On Eden ready time seems to be around ~5s, if GoH goes off theres another static 5s pause where cerb just idles around. 

GoH ready time should be faster, around 3.1s. Also cerb shouldnt idle after GoH goes off unless its on a fresh AA-round .

Time between cerb ready animation frame starts and HP bars drop appears to be 3.1s on retail

Taken from xabats vid:
https://www.youtube.com/watch?v=Gbaagm61LzM
gohs are for example at 21:41 and 23:32

To be fair though............ we are all boomers. Peak reaction speed is at ~24 years old, upper end of reaction speed decay is 10ms/year, 3,5s delay would be a pretty decent compromise?

**Comment by Steam-arch** [comment]
Mozzy was saying at some point way back it was around 1,5s ready time, might have just been set wrong too/also? Not sure how many times its been adjusted, could be changed to 3.1-3.5s at some point imo though, whenever GoH gets looked at in general.

Also happy birthday aether!

**Comment by setsuk0** [comment]
Three years ago, both gates and fulmination were insta-cast. I believe it was Zeta who did the work to change it (might be wrong), but when it was fixed then, we had looked up videos on retail cast time for gates/fulm. If I remember correctly, it might have been Selphius who had done some retail testing and posted some retail videos, but I don't have links to those atm. But those were what the changes were originally based off of.

2008 video-
https://youtu.be/UQSmb8bEgIM?t=36

2009 video- https://youtu.be/t4IgygjDsno?t=62

2007 video https://youtu.be/aA1c6Qb16g8?t=294

From what I can tell in the above videos with gates going off, it does seem to be slightly over 3 seconds.
